src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -1,10 +1,20 @@
 import { Injectable } from '@angular/core';
-import { DotcmsConfig } from 'dotcms-js/dotcms-js';
+import { RequestMethod } from '@angular/http';
+import { CoreWebService } from 'dotcms-js/dotcms-js';
 import { Observable } from 'rxjs/Observable';
 
+/**
+ * Handle license information of current logged in user
+ * @export
+ * @class DotLicenseService
+ */
 @Injectable()
 export class DotLicenseService {
-    constructor(private dotcmsConfig: DotcmsConfig) {}
+    private configUrl: string;
```

Comment:
I think here is better a name like: licenseURL, after all we should change this url by the license end point url when we'll have it